### PR TITLE
Add table of recent Tocks to user details

### PR DESCRIPTION
**Pull request by Jkrzy**
## Description

MVP for #900

For each user details page, we display a table of the 5 most  recent reporting periods and hours per project tock'd therein. 

Table is rendered on both administrative and standard views of user details page.

## Additional information
**Example using fake data**

Screen Shot 2019-11-25 at 11 37 32 AM

### CODE REVIEW

File: tock/employees/views.py [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+from django.conf import settings
 from django.contrib.auth.models import User
 from django.urls import reverse
-from django.views.generic import ListView, DetailView
+from django.views.generic import DetailView, ListView
 from django.views.generic.edit import FormView
 from rest_framework.permissions import IsAuthenticated
-
-from tock.utils import PermissionMixin, IsSuperUserOrSelf
+from tock.utils import IsSuperUserOrSelf, PermissionMixin
```
Comment (tbaxter-18f) [inline tock/employees/views.py]: Just curious  -- and unrelated to this PR --  are you using an opinionated linter (black or whatever)? Should we recommend developing with one?

File: tock/tock/templates/employees/user_recent_tocks.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        >
+            {{tc.reporting_period.start_date}}
+        </a>
+    </th>
+
+    {% endfor %}
+  </thead>
+  <tbody>
+
+  </tbody>
```
Comment (tbaxter-18f) [inline tock/tock/templates/employees/user_recent_tocks.html]: I think you closed your tbody prematurely.

Comment (Jkrzy) [reply tock/tock/templates/employees/user_recent_tocks.html]: Ah thanks!

Amended w/ correction

File: tock/employees/views.py (hunk 1 above)
Comment (Jkrzy) [reply tock/employees/views.py]: Not beyond the established ones, Flake8 and bandit which are in the docs and CI pipeline. This was me running a sort after adding an import.

That said...I'm not sure if Flake8 is scanning for https://www.python.org/dev/peps/pep-0008/#imports at the moment. Will look into it.

**Pull request merged**